File: PHP/random-gen-sec.php
<?php
$n = 1000;
$samples = 1000;

for($s = 0 ; $s < $samples; $s++){
	$array = array();
	for($i = 0 ; $i < $n; $i++){
		array_push($array, random_int(0,PHP_INT_MAX)/PHP_INT_MAX);
	}

	$name = "./training/sec-samples/php-rand-sec".($s+1).".json";
	$file = fopen($name, "w");
	fwrite($file, json_encode($array));
}

File: Python/random-gen-sec.py
# Noted as completely deterministic 
# and not suitable for cryptographic purposes
import json
from numpy.random import default_rng

# Number of gathered items
n = 1000
samples = 500

for s in range(samples):
    arr = []

    name = "./sec-samples/"+"python-rand-sec"+str(s+1)+".json"
    # Initialize generator. 
    # We initialize it so it has a consistent seed
    rand = default_rng()

    #collect data
    for i in range(n):
        num = rand.random()
        arr.append(num)

    #print(arr)
    # Convert to JSON obj
    json_object = json.dumps(arr) 

    with open(name, "w") as outfile:
        outfile.write(json_object)